packages/zipkin/index.d.ts

```diff
@@ -36,6 +36,7 @@ declare namespace zipkin {
     scoped<V>(callback: () => V): V;
     local<V>(name: string, callback: () => V): V;
     createRootId(): TraceId;
+    createIdFromSamplingFlags(isSampled: option.IOption, isDebug: number): TraceId
```

Comment:
this api seems a little awkward, specifically the number thing. It seems a subtype of createRootId really

could this be done something like this?

createRootId(isSampled: option.IOption  = None, isDebug: boolean = false)

Reply:
That was my first idea but the final code had quite some `if`s. Will give a
second try.

Reply:
WDYT @adriancole https://github.com/openzipkin/zipkin-js/pull/247/files#diff-fda6f91886ead20666c649327e0fe1e8R38